sections/algorithmic_components.tex

```diff
@@ -795,41 +790,59 @@ \subsection{Retrieve Diffim Template for a Visit}
-
-Detection on the difference image occurs through correlation of $D(x,y)$ with the science image's PSF, yielding optimally filtered detection image $D'(x,y) = D(x,y) \circ PSF_S(u,v;x,y)$ where $\circ$ denotes correlation (currently the DM stack uses convolution instead of correlation). The values of the pixels in $D'(x,y)$ provide a maximum likelihood estimate of there being a point source at that position. Detection occurs by simply finding pixels that are more than $N \times \sigma$ above the square root of the per--pixel variance. Alternatively, the science image may be {\em pre-convolved} with a kernel similar to its PSF, in which case the resulting difference image $D(x,y)$ is already filtered for detection. This latter option has the additional advantage that it help us avoid requiring {em deconvolution} of the template image, in cases when the template has a wider PSF than the science image. 
+The mechanics of image subtraction will depend on the choice of algorithm.  The performance of several variations are being considered \citep{Reiss16}.  The algorithms differ primarily in their treatment of correlated noise and whether PSFs are measured directly or a matching kernel is used.
+
+In the classic method of Alard \& Lupton (A\&L) \citep{Alard98}, the
+science image is modeled as a convolution of the template image by a
+PSF--matching kernel $\kappa(u,v;x,y)$, i.e., $S = \kappa \otimes
+T$.\footnote{Indices $u,v$ indicate that the kernel itself is a
+2--dimensional function, which varies as a function of position $x,y$ in
+the image; during convolution and correlation there is an implicit
+summation over $u,v$.} Then the difference image, upon which new or
+variable sources are detected, is given by $D = S - (\kappa \otimes
+T)$.
+We model the PSF--matching kernel by decomposing it into a set of basis
+functions $\kappa(u,v) = \sum_i a_i \kappa_i(u,v)$, where the coefficients
+are determined via ordinary least-squares estimation.  The basis functions
+$\kappa_i(u,v)$ are a degree of freedom in this problem.
+\footnote{The current implementation of the A\&L matching algorithm is
+summarized in detail by Becker et al. (2013) (\url{http://ls.st/x9f}).}
```

Comment:
This should be a proper reference to the DocuShare document. I note that in this case the URL actually points to a trac attachment rather than the real DocuShare location which should be LDM-227.